<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_cargarsenal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar se침al</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>canal:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seleccion_canal">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar canal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_atras">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_adelante">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Desplazamiento en el  tiempo:</string>
    </property>
   </widget>
   <widget class="QWidget" name="campo_graficacion" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>541</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="campo_filtrada" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>521</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione los siguientes parametros para el an치lisis espectral:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformada de Wavelet continua</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_atras2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_adelante2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>360</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Desplazamiento </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione tipo de Ventana:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tipo_ventana">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>620</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione el metodo de an치lisis espectral:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodo Welch</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Multitaper</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione los parametros para el an치lisis espectral : :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>660</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frecuencia de Muestreo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>660</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
